### DISCUSSION TITLE
Survey about your Mobile Tech Stack

### DISCUSSION THREAD

**Discussion by wu-sheng** [apache/skywalking#10292] [Polls]
In 2023, the SkyWalking community is seeking the opportunity to add mobile monitoring support(tracing and metrics). 
We have to ask for which tech stack our users are interested in most.

**Comment by witskeeper** [comment]
MAUI

**Reply by wu-sheng** [reply]
https://github.com/dotnet/maui

**Comment by lujiajing1126** [comment]
Due to historic reasons, we are using https://github.com/alibaba/weex, previously an Apache Top-Level project. The container is embedded in the native app.

And also we have a pure react-native app for internal employees.

P.S. Last year, we've designed a private protobuf-based binary protocol based on [sentry v7 event structure](https://develop.sentry.dev/sdk/event-payloads/types/) for both browser and native applications. And mobile developers can still keep using those well-fledged sentry SDKs as much as possible while only the network component should be replaced.

**Reply by wu-sheng** [reply]
weex was an incubator project, but fails in the incubating process.

**And also we have a pure react-native app for internal employees.** Good to know

> P.S. Last year, we've designed a private protobuf-based binary protocol based on [sentry v7 event structure](https://develop.sentry.dev/sdk/event-payloads/types/) for both browser and native applications. And mobile developers can still keep using those well-fledged sentry SDKs as much as possible while only the network component should be replaced.

Sentry is good too.

**Comment by baixjian** [comment]
react-native@0.63.2
